FirstAnalysis/efficiency_studies.py

```diff
@@ -1,22 +1,78 @@
 #!/usr/bin/env python
-from ROOT import TH1F, TCanvas, TEfficiency, TFile, TLegend, gPad, gStyle
 
+# pylint: disable=missing-module-docstring, missing-function-docstring
```

Comment:
Is this needed?

Reply:
Does MegaLinter check it?
I ran over the file with pylint as local MegaLinter is complicated

Reply:
MegaLinter runs pylint, so yes, it does check it.

Reply:
can then these pylints comments remain?
Or do you want me to write some docstrings?

Reply:
Every script should have at least some basic documentation at the top, describing what the code does. So it would be nice if you can add a few lines.